tests/sqlparser_bigquery.rs

```diff
@@ -615,8 +615,6 @@ fn parse_table_identifiers() {
         assert!(bigquery().parse_sql_statements(&sql).is_err());
     }
 
-    test_table_ident("da-sh-es", None, vec![Ident::new("da-sh-es")]);
```

Comment:
Why was this removed?

Reply:
It's in the description: 

From what I see in BigQuery docs, da-sh-es is invalid identifier when used without quotes.
https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#unquoted_identifiers

Reply:
```
Unquoted identifiers
 - Must begin with a letter or an underscore (_) character.
 - Subsequent characters can be letters, numbers, or underscores (_).
```
So the existing code was wrong, as `da-sh-es` is `da - sh - es`